lib/runtime/dart_library.js

```diff
@@ -5,7 +5,7 @@
 /* This file defines the module loader for the dart runtime.
 */
 
-var dart_library;
+var dart_library = typeof module !== "undefined" ? module.exports : undefined;
```

Comment:
I've folded things the other way around for now: `var dart_library = typeof module != "undefined" && module.exports || {};`, keeping nicer module patterns as a snack for later if that's okay :-)